### PR TITLE
Add new level - "Impossible" + More verbose docs

### PR DESCRIPTION
**Pull request by g0tmi1k**
**...this will change DVWA as people know it** _(hopefully for the best!)_!

+ **What was "high" level is now the "Impossible" level**.
+ Made up a new **high level** for each module _(which is designed to be **hackable**)_.
+ Fixed any bugs in "Impossible", which would allow for people to hack it _(when they shouldn't of)!_
+ More verbose documentation.
+ _General bug fixes & code cleanup._

### CODE REVIEW

**Comment by ethicalhack3r** [comment]
:+1:

**Comment by g0tmi1k** [comment]
Wooooooooohoo! :cake:

**Comment by ethicalhack3r** [comment]
:dancers:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,28 @@
-DAMN VULNERABLE WEB APP
+DAMN VULNERABLE WEB APPLICATION
 =======================
 
-v1.8
+v1.9 (*Not Yet Released*)
```
Comment (erwanlr) [inline CHANGELOG.md]: Shouldn't be 1.0.9 ? :p

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,28 @@
...
+Fixed bug with File Include page not loading. (g0tmi1k)
+Fixed CAPTCHA bug where the form wouldn't be visible. (g0tmi1k)
+Fixed CAPTCHA bug where the URL parameters were not being used for low + medium. (g0tmi1k)
+Removed PostgreSQL DB support. (g0tmi1k)
+Renamed 'high' level to 'impossible' and created new vectors for 'high'. (g0tmi1k)
+Updated README and documentation. (g0tmi1k)
+Various code cleanups in the core PHP files + CSS. Verbosed the documentation. (g0tmi1k)
+Various setup improvements (e.g. redirection + limited menu links). (g0tmi1k)
+
+v1.8 (2013-05-01)
```
Comment (erwanlr) [inline CHANGELOG.md]: and 1.0.8 there ?

File: CHANGELOG.md (hunk 1 above)
Comment (g0tmi1k) [inline CHANGELOG.md]: Note based on your last commit:  
`+Versioning change: Version numbers now follow Major.Minor (e.g. 1.8) removing the middle digit.`
Source: https://github.com/RandomStorm/DVWA/commit/414650738add48edac98da5a85a4365b0edfdc72#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR7

File: CHANGELOG.md (hunk 2 above)
Comment (g0tmi1k) [inline CHANGELOG.md]: See the first line for v1.8...

**Comment by g0tmi1k** [comment]
Will soon be a new PR with a few bug fixes (ETA tomorrow).

**Pull request merged**